build.sbt

```diff
@@ -31,18 +27,19 @@ val commonSettings: Seq[Setting[_]] = Seq(
       "scm:git:git@github.com:scala-js/scala-js-env-selenium.git",
       Some("scm:git:git@github.com:scala-js/scala-js-env-selenium.git"))),
   testOptions += Tests.Argument(TestFramework("com.novocode.junit.JUnitFramework"), "-v", "-a")
-) ++ mimaDefaultSettings
+)
 
-val previousArtifactSetting: Setting[_] = {
-  previousArtifact := {
+val previousArtifactSetting = Def.settings()
+  /* MiMa is completely disabled while we are in 1.0.0-SNAPSHOT.
+  mimaPreviousArtifacts ++= {
     val scalaV = scalaVersion.value
     val scalaBinaryV = scalaBinaryVersion.value
     if (!scalaVersionsUsedForPublishing.contains(scalaV)) {
```

Comment:
Since you removed `scalaVersionsUsedForPublishing`, perhaps also remove this test?